src/test/java/com/google/devtools/build/lib/remote/RemoteSpawnCacheTest.java

```diff
@@ -205,6 +205,29 @@ public void injectDigest(ActionInput output, FileStatus statNoFollow, byte[] dig
         }
       };
 
+  private SimpleSpawn simpleSpawnWithExecutionInfo(ImmutableMap<String, String> executionInfo) {
+    return new SimpleSpawn(
+        new FakeOwner("Mnemonic", "Progress Message"),
+        ImmutableList.of("/bin/echo", "Hi!"),
+        ImmutableMap.of("VARIABLE", "value"),
+        executionInfo,
+        ImmutableList.of(ActionInputHelper.fromPath("input")),
```

Comment:
nit: `/* inputs= */ ImmutableList.of(ActionInputHelper.fromPath("input")),`

we add this /* inputs= */ commenst to make it easier ti read, when you are not reading the code from Intellij or other Idea (github, for example)